packages/sdk/src/lib/dataProtectorSharing/subgraph/getCollectionById.ts

```diff
@@ -22,6 +24,12 @@ export async function getCollectionById({
           duration
           price
         }
+        subscriptions(orderBy: endDate, orderDirection: desc, where: { endDate_gte: "${today}" }){
+          endDate
+          subscriber {
+            id
+          }
+        }
```

Comment:
See if that's really necessary once checks before consuming have been finalized.